src/collective/collectionfilter/interfaces.py

```diff
@@ -1,18 +1,17 @@
 # -*- coding: utf-8 -*-
-from collective.collectionfilter import PLONE_VERSION
-from collective.collectionfilter import _
-from collective.collectionfilter import utils
 from plone.api.portal import get_registry_record as getrec
 from plone.app.z3cform.widget import RelatedItemsFieldWidget
 from plone.autoform.directives import widget
+from z3c.form import validator
 from zope import schema
 from zope.interface import Interface
 from zope.publisher.interfaces.browser import IDefaultBrowserLayer
 
+from collective.collectionfilter import PLONE_VERSION, _, utils
 
 try:
-    from plone.formwidget.geolocation.vocabularies import default_map_layer
-    from plone.formwidget.geolocation.vocabularies import default_map_layers
+    from plone.formwidget.geolocation.vocabularies import (default_map_layer,
+                                                           default_map_layers)
```

Comment:
Isn't the previous form the one favorited by our code quaility settings?
See: https://github.com/plone/bobtemplates.plone/blob/master/setup.cfg#L20
and also: https://github.com/plone/Products.CMFPlone/blob/master/setup.cfg#L9

Reply:
that was isort without setup.cfg - already changed that locally